static/app/components/events/interfaces/spans/spanSiblingGroupBar.tsx

```diff
@@ -55,12 +55,22 @@ export default function SpanSiblingGroupBar(props: Props) {
     const operation = spanGrouping[0].span.op;
     const description = spanGrouping[0].span.description;
 
+    if (!operation && !description) {
+      return <strong>{`${t('Autogrouped')} \u2014 ${t('siblings')}`}</strong>;
+    }
+
+    if (!operation) {
+      return <strong>{`${t('Autogrouped')} \u2014 ${description}`}</strong>;
+    }
+
+    if (!description) {
+      return <strong>{`${t('Autogrouped')} \u2014 ${operation}`}</strong>;
+    }
```

Comment:
@0Calories This might not be a big deal, but these if conditions might be more readable when the execution flow is as follows:

```js
if (description) {
    return <strong>{`${t('Autogrouped')} \u2014 ${description}`}</strong>;
}

if (operation) {
    return <strong>{`${t('Autogrouped')} \u2014 ${operation}`}</strong>;
}

return <strong>{`${t('Autogrouped')} \u2014 ${t('siblings')}`}</strong>;
```

Reply:
Definitely more readable this way 😄